src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -225,6 +233,17 @@ public unsafe static bool TryGetPublicKey(ImmutableArray<byte> blob, out Immutab
                 // The RSA magic key must match the blob id
                 if (type == PrivateKeyBlobId && magic == RSA2)
                 {
+                    RSAParameters privKey = default;
```

Comment:
[It is suggested](https://github.com/dotnet/corefx/pull/18024#issuecomment-321422312) that I drop leading zeros, so I'll do that here.